docs/user_manual/selecting.md

```diff
@@ -40,6 +40,13 @@ If you want to delete the entire selection, the easiest way is to deselect every
 ### Inverting Selection
 Inverting the selection means that all of the currently selected pixels will get deselected, while all of the unselected pixels will get selected. You can do this by either pressing <kbd>Control + I</kbd>, or going to the Select menu and pressing **Invert**. If you have everything selected, inverting will clear the selection. Likewise, if you have nothing selecting, inverting will select everything.
 
+## Duplicating
+If you **don't** have an active selection, holding <kbd>Control + Alt + Mouse click</kbd> will *duplicate* whatever was below it and place it in active selection.
+
+:::tip Duplicating Quickly!
+**During active selection** <kbd>Control + Alt + Mouse click</kbd> will not only apply selection but will also **retain** it's contents, which you can use again. Handy when duplicating something multiple times.
```

Comment:
Same as above, I recommend changing "active selection" to "active transformation" and "apply selection" to "[apply transformation](https://orama-interactive.github.io/Pixelorama-Docs/user_manual/transforming#applying-a-transformation)", as well as "it's contents" to "its content".

Reply:
done!